eos-phone-home

```diff
@@ -60,9 +60,15 @@ import requests
 
 log = logging.getLogger(__name__)
 
+Config = collections.namedtuple(
+    'Config',
+    ['host', 'debug', 'force', 'exit_on_server_error'],
+)
 Metrics = collections.namedtuple('Metrics', ['enabled', 'environment'])
 ProductInfo = collections.namedtuple('ProductInfo', ['vendor', 'product'])
 
+DEFAULT_CONFIG_PATH = '/etc/metrics/eos-phone-home.conf'
```

Comment:
I don't especially like this path because this component is (on the client side) conceptually not part of the metrics system. Of course I accept the reality that the data goes into the same database on the server.

Could it just be `/etc/eos-phone-home.conf`?

Reply:
Or for symmetry with the state in `/var/lib/eos-phone-home`, `/etc/eos-phone-home/eos-phone-home.conf`, pleading forgiveness for the repetition?

Reply:
I'm fine with `/etc/eos-phone-home.conf`. I don't imagine we'd need more than a single file that would necessitate a directory.

Reply:
I went with `/etc/eos-phone-home.conf`.